### PR TITLE
No more references to ONI

### PR DESCRIPTION
**Pull request by daortizh**
- Updated code references to ONI
- Updated documentation references to ONI

### CODE REVIEW

File: spot-oa/ui/ipython/custom/custom.css [hunk 1]
```diff
@@ -65,81 +65,81 @@
...
-.oni.oni_easy_mode:not(.oni_easy_mode_loading) #oni_easy_mode_loader
+.spot.spot_easy_mode:not(.spot_easy_mode_loading) #spot_easy_mode_loader
 {
     right: 25px;
     margin-top: 0px;
     width: auto;
 }
 
-.oni.oni_ninja_mode #oni_easy_mode_loader
+.spot.spot_ninja_mode #spot_easy_mode_loader
```
Comment (natedogs911) [inline spot-oa/ui/ipython/custom/custom.css:75]: in the future we may think about changing ninja to something more transparent like "advanced"

File: spot-oa/ui/package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "oni_ui",
+  "name": "spot_ui",
   "version": "1.1.0",
   "description": "Open Network Insight :: UI",
```
Comment (natedogs911) [inline spot-oa/ui/package.json]: should this be changed to Apache Spot?

File: spot-oa/ui/proxy/package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "oni_ui_proxy",
+  "name": "spot_ui_proxy",
   "version": "1.1.0",
   "description": "Open Network Insight :: UI :: Proxy",
```
Comment (natedogs911) [inline spot-oa/ui/proxy/package.json]: Open Network Insight --> Apache Spot

**Review by natedogs911 — COMMENTED** [review]
Please check the few notes i have provided

File: spot-oa/ui/ipython/custom/custom.css (hunk 1 above)
Comment (daortizh) [reply spot-oa/ui/ipython/custom/custom.css:75]: Sure, we can discuss that. I used ninja as in "CSS ninja" or "JavaScript ninja", those terms refer to "advanced" knowledge/usage of a technology. Thanks

**Comment by LedaLima** [comment]
LGTM 👍 
Long Live Spot

**Pull request merged**